### ISSUE TITLE
New tenant can't log in on first ever Peregrine launch

### ISSUE THREAD

**Issue by reggie7** [headwirecom/peregrine-cms#661]
**Steps to Reproduce**
1. Build a new launcher from under `sling/peregrine-builder-sling-12`,
1. Start a perfectly clean instance based on the above,
1. Log in as admin,
1. Create a new Site / Tenant with a password,
1. Log out and log in as the new tenant user.

**Expected Result**: You can do it, you're in.
**Actual Result**: The page is blank, `/index.html` results in a `404`, the mapping from `/content/admin/pages/index.html` to `/admin/pages/index.html` does not work.

**Comment by reggie7** [comment]
I know there is a simple resolution to this: just restart the instance. But it's not the goal that I think we should aim at. It's at least detrimental to the developer experience - I just wasted some time on this as I like to sometimes get a clean instance and that's the first time when I have tried such a sequence. We should at least print a message in the UI to ask the user to restart the system after the first run. It all boils down to first impressions.
